aptos-move/e2e-move-tests/tests/code_publishing.rs

```diff
@@ -221,6 +221,7 @@ fn code_publishing_framework_upgrade_fail() {
     assert_vm_status!(result, StatusCode::BACKWARD_INCOMPATIBLE_MODULE_UPDATE)
 }
 
+#[ignore]
```

Comment:
So why do we disable these tests?

Reply:
Because these tests test the arbitrary upgrade feature, which has been disabled.